packages/ui/components/form/dropdown/Dropdown.tsx

```diff
@@ -9,7 +9,7 @@ import { ButtonColor } from "@calcom/ui";
 
 export const Dropdown = DropdownMenuPrimitive.Root;
 
-type DropdownMenuTriggerProps = ComponentProps<typeof DropdownMenuPrimitive["Trigger"]>;
+type DropdownMenuTriggerProps = ComponentProps<(typeof DropdownMenuPrimitive)["Trigger"]>;
```

Comment:
Did eslint rules changed?

Reply:
I know right, I'm getting the complete opposite locally. Not sure what to do.